src/CentreonLegacy/Core/Utils/Utils.php

```diff
@@ -212,9 +212,6 @@ public function encodePass($password, $algo = 'md5')
             case 'md5':
                 $encodePassword .= 'md5__' . md5($password);
                 break;
-            case 'sha1':
-                $encodePassword .= 'sha1__' . sha1($password);
-                break;
```

Comment:
You should also use BCRYPT as default case.

Reply:
@jeremyjaouen Oki it's done, just a quick note while i was testing i realised even when it was like this, passwords were stored as Bcrypt  but i dunno if this is relevant ^^.